RELEASE_NOTES.md

```diff
@@ -12,9 +12,9 @@
 - Creating configurations just became easier. Using a DSL you can now create configurations in a more declarative and concise way:
 ```Kotlin
 CheckoutConfiguration(
+    shopperLocale = shopperLocale,
```

Comment:
I had moved `shopperLocale` under `environment` and `clientKey` on purpose because it is now optional

Reply:
removed that line so doesn't apply. resolving the conversation.

Reply:
moved it below `environment` and `clientKey`